src/components/Editor/Editor.css

```diff
@@ -30,9 +30,9 @@
  */
 .editor-wrapper {
   position: absolute;
-  height: calc(100% - 29px);
+  height: calc(100% - 30px);
```

Comment:
this seems like a magic number, maybe lets make this a var?

Reply:
cool